megabus/src/main/java/com/bazaarvoice/megabus/MegabusRef.java

```diff
@@ -26,14 +26,17 @@
     private final String _key;
     private final UUID _changeId;
     private final Instant _readTime;
+    private final boolean _deleted;
```

Comment:
consider to switch to `enum` with states

Reply:
what is the purpose for this? To have flexibility in the future?

Reply:
yep